spec/basket_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'basket'
+
+describe Basket do
+
+  context 'Describe review_order:' do
+    before do
+      subject.basket["Satay Chicken"] = 6
+    end
+    it 'returns the item in the basket and the total cost' do
+      expect(subject.review_order).to eq "Satay Chicken, £6\nTotal: £6"
+    end
+  end
+
+end
```

Comment:
As the `total` and `view` methods are public, they should have their own explicit tests